Bare bones running with ERR_MODULE_NOT_FOUND  - looking for keypair.js.js

**Node.js v20.10.0**


Error :

node:internal/modules/esm/resolve:255
    throw new ERR_MODULE_NOT_FOUND(
          ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/stevensanborn/Documents/Crypto/devcourse/test/node_modules/@solana-developers/helpers/dist/esm/lib/keypair.js.js' imported from /Users/stevensanborn/Documents/Crypto/devcourse/test/node_modules/@solana-developers/helpers/dist/esm/index.js
    at finalizeResolution (node:internal/modules/esm/resolve:255:11)
    at moduleResolve (node:internal/modules/esm/resolve:908:10)
    at defaultResolve (node:internal/modules/esm/resolve:1121:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:396:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:365:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:240:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)
    at link (node:internal/modules/esm/module_job:84:36) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///Users/stevensanborn/Documents/Crypto/devcourse/test/node_modules/@solana-developers/helpers/dist/esm/lib/keypair.js.js'

here is what im doing 


`mkdir test
cd test
npm i @solana-developers/helpers
npx esrun main.ts
`

main.ts
` import { getKeypairFromEnvironment } from "@solana-developers/helpers";
import 'dotenv/config';
console.log(getKeypairFromEnvironment("SECRET_KEY"));`

--- a/post-build.mjs
+++ b/post-build.mjs
@@ -34,12 +34,14 @@ async function processEsmFiles(dir) {
       } else if (entry.isFile() && entry.name.endsWith('.js')) {
         let content = await fs.readFile(fullPath, 'utf8');
         
-        // Add .js extension to import statements
-        content = content.replace(/from\s+['"](\.[^'"]+)['"]/g, "from '$1.js'");
-        
-        // Add .js extension to export statements
-        content = content.replace(/export\s+\*\s+from\s+['"](\.[^'"]+)['"]/g, "export * from '$1.js'");
-        
+        // Replace relative imports and exports with .js extension
+        // import or export, followed by optional whitespace, optional from, then optional whitespace
+        // skip .js if already present
+        const regex = /(import|export)(\s+[\s\S]*?\s+(?:from)?\s+['"])(\.\/[^'"]+)(['"])/g;
+        content = content.replace(regex, (match, keyword, beforePath, path, quote) => {
+          return `${keyword}${beforePath}${path}${path.endsWith('.js') ? '' : '.js'}${quote}`;
+        });
+
         await fs.writeFile(fullPath, content);
         console.log(`Processed ESM file: ${fullPath}`);
       }

